client/packages/common/src/types/schema.ts

```diff
@@ -3416,7 +3408,7 @@ export type Queries = {
   invoiceCounts: InvoiceCounts;
   invoices: InvoicesResponse;
   itemCounts: ItemCounts;
-  itemVariantsList: Array<ItemVariantNode>;
+  itemVariantsList: Array<UnitVariantNode>;
```

Comment:
There is a bit of renaming like this, thought to simplify it to just UnitVariant for now, how does that sound ?

Reply:
hm. both  `UnitVariantNode` and `ItemVariantNode` are a hard to know what they mean, unless you are already working with them.

Reply:
Yes I agree, can we solve it with better naming, or overall dev docs on the subject ? I really not sure, I am adding a link to this comment to  https://github.com/openmsupply/open-msupply/issues/2078

Reply:
I think stick with UnitVariantNode - it's making more sense now that I understand a little